### PR TITLE
Update OnInspectorGUI for Toolkit Inspector

### PR DESCRIPTION
**Pull request by Proton-V**
## Changes
- Fixes: #10720

## Overview
Fixed micro issue with MixedRealityToolkit Inspector
Issue: error stack when we have active MixedRealityToolkit Inspector in editor and active profile is changed

## Fixes
- OnInspectorGUI method in MixedRealityToolkitInspector.cs

### CODE REVIEW

**Comment by MaxWang-MS** [comment]
Hi @Proton-V, thanks for opening the PR. I think by adding the `if` statement you are limiting the ability to switch profiles in the inspector to only when the application is not in play mode. I would propose adding an `else` statement containing the following call:
`MixedRealityToolkit.Instance.ActiveProfile = (MixedRealityToolkitConfigurationProfile)activeProfile.objectReferenceValue;`
This way one can also switch profiles in the inspector while the application is in play mode.

**Comment by Proton-V** [comment]
@MaxWang-MS Hello. Of course, you're right, thanks!

File: Assets/MRTK/Core/Inspectors/MixedRealityToolkitInspector.cs [hunk 1]
```diff
@@ -79,6 +77,29 @@ public override void OnInspectorGUI()
...
+                }
+                else
+                {
+                    MixedRealityToolkit.Instance.ActiveProfile = newProfile;
+                }
+
+                activeProfileEditor = null;
+                cachedProfile = activeProfile.objectReferenceValue;
+            }
+            catch(System.Exception e)
```
Comment (MaxWang-MS) [inline Assets/MRTK/Core/Inspectors/MixedRealityToolkitInspector.cs:97]:
```suggestion
            catch (System.Exception e)
```

**Comment by Proton-V** [comment]
> Thanks for the contribution! Just curious, have you run into situations where an exception was thrown? I am wondering why you added the try catch.

I added this for future changes. 
Possible that these methods may sometimes work incorrectly (later, currently i dont see this behaviour).

And i think handling error without stop action - normal way.

**Comment by MaxWang-MS** [comment]
> > Thanks for the contribution! Just curious, have you run into situations where an exception was thrown? I am wondering why you added the try catch.
> 
> I added this for future changes. Possible that these methods may sometimes work incorrectly (later, currently i dont see this behaviour).
> 
> And i think handling error without stop action - normal way.
> 
> @MaxWang-MS Tell me what do you think about it. I will be glad to hear your thought

I don't have a super strong opinion on this, but in general I tend to use try catch when I expect things to _sometimes_ go wrong. At the moment I don't really expect exceptions to be thrown when the profile change is initiated the correct way, so I was wondering why you added them. I don't think it hurts to keep them here though.

**Pull request merged**